### ISSUE TITLE
what's the difference between  「loss='binary_crossentropy'」 and 「loss=losses.binary_crossentropy」?

### ISSUE THREAD

**Issue by Donald-Su** [keras-team/keras#13125]
In the `model.compile` step, what's the difference between  `loss='binary_crossentropy` and `loss=losses.binary_crossentropy` ?

compile 1:
```
model.compile(optimizer=optimizers.RMSprop(lr=0.001),
           loss='binary_crossentropy, 
           metrics=[metrics.binary_accuracy])
```
compile 2:
```
model.compile(optimizer=optimizers.RMSprop(lr=0.001),
           loss=losses.binary_crossentropy, 
           metrics=[metrics.binary_accuracy])
```

**Comment by gowtham-kp** [comment]
@Donald-Su Please post this question n Stack overflow or Keras Slack channel as this platform is meant for bug/performance, feature request or related issues. Thanks!
